.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+- '2.7'
+- '3.6'
```

Comment:
I would recommend adding 3.5 and 3.4 too, there are low level implementation details that make a difference in some pieces of code, and that way you are covered.